packages/library-base/src/commonMain/kotlin/io/realm/kotlin/MutableRealm.kt

```diff
@@ -126,4 +126,20 @@ public interface MutableRealm : TypedRealm {
      * @throws IllegalArgumentException if the object is invalid, frozen or not managed by Realm.
      */
     public fun delete(deleteable: Deleteable)
+
+    /**
+     * Deletes all objects from this Realm.
+     */
+    public fun deleteAll()
+
+    /**
+     * Deletes all objects of the specified class from the Realm.
+     *
+     * @param KClass the class whose objects should be removed.
+     */
+    public fun delete(schemaClass: KClass<out BaseRealmObject>)
```

Comment:
What happens if you provide a class that doesn't exist? I assume we throw an IllegalArgumentException. In that case we should add a @throws to the Dokka.